al_aws_collector.js

```diff
@@ -704,7 +718,19 @@ class AlAwsCollector {
                 if (err) {
                     return callback(err);
                 } else {
-                    return collector.send(payload, false, ingestType,  callback);
+                    // send the lmc stats if ingest type is logmsgs
+                    collector.send(payload, false, ingestType, (err, res) => {
```

Comment:
I think `ingestType` should be always LOGMSGS.
And we can remove `ingestType` from `processLog(messages, formatFun, hostmetaElems, ingestType = '', callback)`